packages/ws-server/src/utils/utils.ts

```diff
@@ -68,35 +69,82 @@ export const getMultipleAddressesEnabled = (): boolean => {
 /**
  * Sends a JSON-RPC response message to the client WebSocket connection.
  * Resets the TTL timer for inactivity on the client connection.
- * @param {any} connection - The WebSocket connection object.
- * @param {any} request - The request object received from the client.
- * @param {any} data - The data to be included in the response.
- * @param {string} tag - A tag used for logging and identifying the message.
- * @param {any} logger - The logger object for logging messages and events.
+ * @param {any} connection - The connection object.
+ * @param {any[]} params - The parameters associated with the request.
+ * @param {any} method - The method associated with the request.
+ * @param {any} response - The response to send to the client.
+ * @param {any} logger - The logger object.
  * @param {string} requestIdPrefix - The prefix for the request ID.
  * @param {string} connectionIdPrefix - The prefix for the connection ID.
  */
 export const sendToClient = (
   connection: any,
-  request: any,
-  data: any,
-  tag: string,
+  params: any,
+  method: any,
+  response: any,
   logger: any,
   requestIdPrefix: string,
   connectionIdPrefix: string,
 ) => {
+  const TX_INFO = params[0];
+  const BLOCK_PARAM = params[1];
+  const TAG = JSON.stringify({ method, txInfo: TX_INFO, block: BLOCK_PARAM });
+
   logger.info(
-    `${connectionIdPrefix} ${requestIdPrefix}: Sending data from tag: ${tag} to connectionId: ${
-      connection.id
-    }, data: ${JSON.stringify(data)}`,
+    `${connectionIdPrefix} ${requestIdPrefix}: Sending data=${JSON.stringify(
+      response.result,
+    )} to client from tag=${TAG}`,
   );
 
-  connection.send(
-    JSON.stringify({
-      jsonrpc: '2.0',
-      result: data,
-      id: request.id,
-    }),
-  );
+  connection.send(JSON.stringify(response));
   connection.limiter.resetInactivityTTLTimer(connection);
 };
+
+/**
+ * Handles sending requests to the relay for processing.
+ * @param {string} tag - The tag associated with the request.
+ * @param {any} args - The arguments to be passed to the relay.
+ * @param {Relay} relay - The relay instance used to process the request.
+ * @param {any} logger - The logger instance used for logging.
+ * @param {string} rpcCallEndpoint - The endpoint on the relay to call.
+ * @param {string} requestIdPrefix - The prefix to use for the request ID.
+ * @param {string} connectionIdPrefix - The prefix to use for the connection ID.
+ * @returns {Promise<any>} A promise that resolves to the result of the request.
+ */
+export const handleSendingRequestsToRelay = async (
+  tag: string,
```

Comment:
why on this method we receive TAG already formed instead of receiving both `method` and `params` and we create the Tag here (similar to sendToClient method above), maybe we can create a method that receives the method and params and returns the TAG, that can be shared and used whenever necessary, so we can explain there the usage of TAG, and also is more descriptive to anyone reading the code for the first time.

Also if we chose to send both method and params instead of the TAG, we could get rid of `args` parameter and call the `resolvedParams` in here, further simplifying the logic a bit.

Reply:
oh this is briliant why didn't I think of it! This would make the code in the server so much cleaner! Thanks so much Fredy!